ui/src/app/stylesheets/_table.scss

```diff
@@ -210,3 +210,7 @@ $table-border-color:            #e0e0e0;
     }
   }
 }
+
+.vertalign{
+  padding-top: 3% !important;
```

Comment:
Make it an id and you can get rid of the `!important`. Also IMO _custom.scss is a better place to have this.